release/_github.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+function invoke_github_api {
+	local curl_response=$(\
+		curl \
+			"https://api.github.com/repos/kiwm/${1}" \
+			--data "${2}" \
+			--header "Accept: application/vnd.github+json" \
+			--header "Authorization: Bearer ${LIFERAY_RELEASE_GITHUB_PAT}" \
+			--header "X-GitHub-Api-Version: 2022-11-28" \
+			--include \
+			--max-time 10 \
+			--request POST \
+			--retry 3 \
+			--silent)
+
+	if [ $(echo "${curl_response}" | awk '/^HTTP/{print $2}') -ne 201 ]
+	then
+		lc_log ERROR "Unable to invoke GitHub API:"
+		lc_log ERROR "${curl_response}"
+
+		return "${LIFERAY_COMMON_EXIT_CODE_SKIPPED}"
+	fi
+}
+
+function invoke_github_api_with_response {
```

Comment:
@kiwm, do we really need to have 2 methods to invoke the GitHub API? Can't we merge these 2 methods?

Reply:
We need the two methods one is a get with an response and the other is a post that dont' have the response.